tutorials/step_by_step/filesystem.rst

```diff
@@ -46,7 +46,7 @@ engine.cfg
 ----------
 
 The engine.cfg file is the project description file, and it is always found at
-the root of the project, in fact it's location defines where the root is. This
+the root of the project. In fact it's location defines where the root is. This
```

Comment:
While fixing English mistakes, it would have been nice to fix this hideous `it's` too :P